index.js

```diff
@@ -1 +1,36 @@
+import { DateTime } from './module/luxon.js';
+import { addBook, getStorageData, displayBooks } from './module/books.js';
+import Library from './module/lists.js';
+import {
+  displayList,
+  displayAdd,
+  displayContact,
+} from './module/navigations.js';
 
+const Listli = document.getElementById('List');
+const Addli = document.getElementById('AddNew');
+const Contactli = document.getElementById('ContactPage');
+
+const library = new Library();
+library.awesomeBooks = [];
+
+window.onbeforeunload = () => {
+  getStorageData(library);
+  displayBooks(library);
+};
+
+const currentTime = DateTime.now().toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+document.getElementById('date').innerHTML = currentTime;
```

Comment:
- [Optional] It would be nice to update the time every second or minute. To achieve this, you can use JavaScript's setInterval function. If you are free kindly read [this article](https://www.w3schools.com/js/js_timing.asp) to give you an idea of how to use it :+1:

Reply:
@FrederickMih Thanks a lot for the review and approval!
I will work on the great optional suggestion provided. 
Cheers!